tests/dummy/Button.h

```diff
@@ -0,0 +1,39 @@
+/**
+ * @file Button.h
+ * @brief ボタンクラス(ダミー)
+ * @author HaruArima08 nishijima515
+ */
+
+#ifndef BUTTON_H
+#define BUTTON_H
+
+namespace spikeapi {
+  class Button {
+   public:
+    /**
+     * コンストラクタ
+     */
+    Button(void) = default;
+
+    /**
+     * LEFTボタンが押されているかを確認する
+     */
+    bool isLeftPressed() const { return isPressedFlag; }
+
+    /**
+     * RIGHTボタンが押されているかを確認する
+     */
+    bool isRightPressed() const { return isPressedFlag; }
```

Comment:
true falseをランダムに返せば実装のwhileつけたままでいいんじゃない？

```suggestion

    // LEFTボタンが押されているかをランダムに判定
    bool isLeftPressed() const {
        return std::rand() % 2 == 0; // 50%の確率でtrueかfalse
    }

    // RIGHTボタンが押されているかをランダムに判定
    bool isRightPressed() const {
        return std::rand() % 2 == 0;
    }
};
```